commit 78863edbb0751f5c9694ea10c6132a87cfd0ee10
Author: Sergei Trofimovich <slyfox@gentoo.org>
Date:   Wed Aug 27 22:20:33 2014 +0300

    Revert "disable shared libs on sparc (linux/solaris) (fixes #8857)"
    
    This reverts commit 623883f1ed0ee11cc925c4590fb09565403fd231.
    
    The commit a93ab43ab5f40cadbedea2f6342b93c245e91434
        driver: pass '-fPIC' option to assembler as well
    fixes shared libraries on sparc at least on linux.
    
    Properly fixes Issue #8857
    
    Signed-off-by: Sergei Trofimovich <slyfox@gentoo.org>

diff --git a/mk/config.mk.in b/mk/config.mk.in
index c210cd3..392237f 100644
--- a/mk/config.mk.in
+++ b/mk/config.mk.in
@@ -97,9 +97,7 @@ endif
 # Some platforms don't support shared libraries
 NoSharedLibsPlatformList = powerpc-unknown-linux \
 	x86_64-unknown-mingw32 \
-	i386-unknown-mingw32 \
-	sparc-sun-solaris2 \
-	sparc-unknown-linux
+	i386-unknown-mingw32
 
 ifeq "$(SOLARIS_BROKEN_SHLD)" "YES"
 NoSharedLibsPlatformList += i386-unknown-solaris2
